tabs/osd.js

```diff
@@ -420,6 +432,12 @@ OSD.chooseFields = function () {
         F.PID_YAW,
         F.POWER
       ]);
+      if (semver.gt(CONFIG.flightControllerVersion, "3.1.0")) {
```

Comment:
That makes sense, initially I'd used FC version for the sake of being consistent as this is what is used to test other OSD features.